canopy.py

```diff
@@ -185,6 +186,55 @@ def convert_afe_to_final_tiles(phyreg_ids):
 
     print('Completed')
 
+def convert_2bit(phyreg_ids):
```

Comment:
`convert_tif_to_2bits`?

Reply:
Also needs a docstring for help.

Reply:
Did not mean to include that function. Was created to explore possibility of converting to 2bit files sooner. I thought I had deleted it out.